lib/rubygems/defaults.rb

```diff
@@ -28,26 +28,26 @@ def self.default_spec_cache_dir
   # specified in the environment
 
   def self.default_dir
-    path = if defined? RUBY_FRAMEWORK_VERSION then
-             [
-               File.dirname(RbConfig::CONFIG['sitedir']),
-               'Gems',
-               RbConfig::CONFIG['ruby_version']
-             ]
-           elsif RbConfig::CONFIG['rubylibprefix'] then
-             [
-              RbConfig::CONFIG['rubylibprefix'],
-              'gems',
-              RbConfig::CONFIG['ruby_version']
-             ]
-           else
-             [
-               RbConfig::CONFIG['libdir'],
-               ruby_engine,
-               'gems',
-               RbConfig::CONFIG['ruby_version']
-             ]
-           end
+    path =  if defined? RUBY_FRAMEWORK_VERSION then
+              [
+                File.dirname(RbConfig::CONFIG['sitedir']),
+                'Gems',
+                RbConfig::CONFIG['ruby_version']
+              ]
+            elsif RbConfig::CONFIG['rubylibprefix'] then
+              [
+                RbConfig::CONFIG['rubylibprefix'],
+                'gems',
+                RbConfig::CONFIG['ruby_version']
```

Comment:
To try to clarify further, you can see in slightly darker green here that you're adding 2 spaces to the previous three lines, and one space to the rest of the lines. Instead it would be enough to add a single space to these three lines and leave the rest as they are. There's actually a cop checking for the offense being introduced here, I think it's `Layout/UnnecessaryWhitespace` or something like that.